pkg/k8shandler/deployment.go

```diff
@@ -343,6 +343,11 @@ func (node *deploymentNode) rollingRestart(upgradeStatus *api.ElasticsearchNodeS
 
 		if replicas > 0 {
 
+			if err := EnforceNetworkPolicy(node.self.Namespace, node.client, node.self.ObjectMeta.OwnerReferences); err != nil {
+				logrus.Warnf("Unable to create network policy: %v", err)
```

Comment:
We should really be logging WHICH cluster this applies given we can support cluster's other then our own.  Comment applies to any log message

Reply:
@jcantrill addressed